cookbooks/Gradio/huggingface.aiconfig.json

```diff
@@ -207,21 +207,20 @@
       "input": {
         "attachments": [
           {
-            "data": "./hi.mp3",
+            "data": "cookbooks/Gradio/hi.mp3",
```

Comment:
For testing I had to change this line becuase I could only run from top level `aiconfig` dir